tests-babel/transform.test.js

```diff
@@ -47,11 +48,11 @@ describe('convert templates', () => {
         some content
       */
       {
-        "id": "xn207nfA",
+        "id": "R0DE3v6/",
         "block": "[[[1,\\"some content\\"]],[],false,[]]",
-        "moduleName": "/rewritten-app/a.hbs",
+        "moduleName": "/tmp/path/my-app/node_modules/.embroider/rewritten-app/components/a.hbs",
```

Comment:
this one still looks goofy

Reply:
mmm, my hope was that it would take the one from templateOnly call

Reply:
Oh, i see why. This will take a bit more